Source/Core/Core/HW/EXI/EXI_DeviceEthernetBase.cpp

```diff
@@ -544,29 +541,29 @@ bool CEXIETHERNET::RecvHandlePacket()
 #endif
 
   // Is the current frame multicast?
-  if (mRecvBuffer[0] & 0x01)
+  if (m_recv_buffer[0] & 0x01)
     status |= DESC_MF;
 
   if (status & DESC_BF)
   {
-    if (mBbaMem[BBA_MISC2] & MISC2_AUTORCVR)
+    if (m_bba_mem.misc2 & MISC2_AUTORCVR)
     {
-      *(u16*)&mBbaMem[BBA_RWP] = rwp_initial;
+      *(u16*)&m_bba_mem.rwp = rwp_initial;
     }
     else
     {
       ERROR_LOG(SP1, "RBF while AUTORCVR == 0!");
     }
   }
 
-  descriptor->set(*(u16*)&mBbaMem[BBA_RWP], 4 + mRecvBufferLength, status);
+  descriptor->set(*(u16*)&m_bba_mem.rwp, 4 + m_recv_buffer_length, status);
```

Comment:
Not completely sure about this one, but I'm guessing it just wants the value of `rwp` (and not a pointer)
```suggestion
  descriptor->set(m_bba_mem.rwp, 4 + m_recv_buffer_length, status);
```